spec/07-variables.md

```diff
@@ -259,6 +281,8 @@ function f()
 }
 ```
 
+Be also aware that declaring a variable global can hide a local variable and/or a function static with the same name. See ([§§](#hidingNotice)) for an example.
```

Comment:
Maybe I am a bad pull request writer, because 3 out of 5 open pull requests are authored by me ;-)... haha
anyway, if I am in a good mood again, and Andrea has explained what she actually means with the statement below, then I might rebase my change and incorporate your feedback